wgengine/magicsock/endpoint.go

```diff
@@ -1069,7 +1069,7 @@ func (de *endpoint) removeSentDiscoPingLocked(txid stun.TxID, sp sentPing, resul
 
 // discoPingSize is the size of a complete disco ping packet, without any padding.
 const discoPingSize = len(disco.Magic) + key.DiscoPublicRawLen + disco.NonceLen +
-	poly1305.TagSize + disco.MessageHeaderLen + disco.PingLen
+	chacha20poly1305.Overhead + disco.MessageHeaderLen + disco.PingLen
```

Comment:
rather than using this package's constant and implying that disco uses chacha20, let's just make a new unexported const local to this file.

Like:

```go
// poly1305AuthenticatorSize is the size, in bytes, of a poly1305 authenticator.
// It's the same as golang.org/x/crypto/poly1305.TagSize, but that
// page is deprecated and we only need this one constant, so we copy it.
const poly1305AuthenticatorSize = 16
```